sent upstream:
https://lists.alioth.debian.org/pipermail/sane-devel/2015-July/033601.html

From 867c8fcde5cf54afae7f9ae8f855605b891708a7 Mon Sep 17 00:00:00 2001
From: Christopher Brannon <teiresias@gentoo.org>
Date: Wed, 8 Jul 2015 06:37:14 -0400
Subject: [PATCH] add a configure flag to control systemd support

This way people can easily disable systemd support locally.
---
 configure.in | 29 ++++++++++++++++++-----------
 1 file changed, 18 insertions(+), 11 deletions(-)

diff --git a/configure.in b/configure.in
index 059058b..b539cda 100644
--- a/configure.in
+++ b/configure.in
@@ -343,17 +343,24 @@ dnl 	$as_echo "in order to autodetect network scanners in kodakaio."
 dnl 	with_cups="no"
 dnl fi
 
-dnl added by llagendijk 12/7/2012 to detect systemd for saned
-$as_echo_n "Checking for systemd..."
-if test -e /usr/include/systemd/sd-daemon.h ; then
-    AC_DEFINE(HAVE_SYSTEMD, 1, [Is /usr/include/systemd/sd-daemon.h available?])
-    with_systemd="yes"
-    SYSTEMD_LIBS=" -lsystemd-daemon"
-    AC_SUBST(SYSTEMD_LIBS)
-    $as_echo "yes"
-else
-    with_systemd="no"
-    $as_echo "no"
+AC_ARG_WITH(systemd,
+	AC_HELP_STRING([--with-systemd], [enable systemd support @<:@default=yes@:>@]))
+if test "x$with_systemd" != xno ; then
+    dnl added by llagendijk 12/7/2012 to detect systemd for saned
+    $as_echo_n "Checking for systemd..."
+    if test -e /usr/include/systemd/sd-daemon.h ; then
+	AC_DEFINE(HAVE_SYSTEMD, 1, [Is /usr/include/systemd/sd-daemon.h available?])
+	have_systemd="yes"
+	SYSTEMD_LIBS=" -lsystemd-daemon"
+	AC_SUBST(SYSTEMD_LIBS)
+	$as_echo "yes"
+    else
+	have_systemd="no"
+	$as_echo "no"
+	if test "x$with_systemd" = xyes; then
+	    AC_MSG_ERROR([Systemd support was requested but systemd was not found])
+	fi
+    fi
 fi
 
 dnl ***********
-- 
2.4.4

